apps/admin/admin_api/pubspec.lock

```diff
@@ -745,14 +745,22 @@ packages:
       url: "https://pub.dev"
     source: hosted
     version: "3.0.0"
+  shrink:
```

Comment:
For compressing what can be very large json messages in relay stats events. Have observed 93% size reduction for larger messages